src/js/common/data_graphic.js

```diff
@@ -121,7 +121,8 @@ MG.data_graphic = function(args) {
     showActivePoint: true,                      // If enabled show active data point information in chart
     brush: null,                                // add brushing function for this chart. could be set as 'xy', 'x', 'y' to restrict axis
     zoom_target: null,                          // zooming target of brushing function. if not set the default is to zoom the current chart
-    brushing_selection_changed: null            // callback function on brushing. the first parameter are the arguments that correspond to this chart, the second parameter is the range of the selection
+    brushing_selection_changed: null,           // callback function on brushing. the first parameter are the arguments that correspond to this chart, the second parameter is the range of the selection
+    click_to_zoom_out: true                     // the switch of allow click to zoom out
```

Comment:
replace the comment here with: `// if true and the graph is currently zoomed in, clicking on the graph will zoom out`